packages/imager/install

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -x
+set -e
+
+apt-get update -qq
+apt-get install -y libfftw3-dev libX11-dev libtiff-dev
```

Comment:
I checked https://github.com/dahtah/imager for linux install.

if don't need to install libX11-dev for imager, will remove fine.